tests/organizations/test_create_organization.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+from datetime import datetime
+from graphql_relay import to_global_id
+from django.contrib.auth import get_user_model
+
+from creator.tasks import setup_cavatica_task
+from creator.organizations.models import Organization
+from creator.organizations.factories import OrganizationFactory
+
+User = get_user_model()
+
+
+CREATE_ORGANIZATION_MUTATION = """
+mutation newOrganization($input: CreateOrganizationInput!) {
+    createOrganization(input: $input) {
+        organization {
+            name
+            description
+            createdOn
+            website
+            email
+            members { edges { node { id username } } }
+        }
+    }
+}
+"""
+
+
+def test_create_organization_mutation(db, permission_client):
+    """
+    Only admins should be allowed to create studies
+    """
+    user, client = permission_client(["add_organization"])
+
+    variables = {"input": {"name": "My Test DCC"}}
+    resp = client.post(
+        "/graphql",
+        content_type="application/json",
+        data={"query": CREATE_ORGANIZATION_MUTATION, "variables": variables},
+    )
+
+    resp_data = resp.json()["data"]["createOrganization"]["organization"]
+    assert resp_data["name"] == "My Test DCC"
+    assert Organization.objects.filter(name="My Test DCC").exists()
+    assert (
+        Organization.objects.filter(name="My Test DCC").get().members.count()
+        == 1
+    )
+
+
+def test_create_organization_mutation_not_allowed(db, permission_client):
+    """
+    Only admins should be allowed to create studies
+    """
+    user, client = permission_client([])
+
+    variables = {"input": {"name": "My Test DCC"}}
+    resp = client.post(
+        "/graphql",
+        content_type="application/json",
+        data={"query": CREATE_ORGANIZATION_MUTATION, "variables": variables},
+    )
+
+    message = resp.json()["errors"][0]["message"]
+    assert "Not allowed" in message
+
+
+def test_create_organization_mutation_invalid_url(db, permission_client):
```

Comment:
I could although I believe it's just a stylistic preference. I am choosing to do it in two tests because we may quickly identify which validator is breaking based on the name of the test that fails.
These tests shouldn't really be necessary, imo. They are only necessary because there's a bit of hacking high up in the resolver that catches `ValidationError` and injects it into the graphql error response and I don't think it's formally tested anywhere. If it were tested at a more general level, we should be able to assume that it's functioning and skip these specific manifestations of it.